python/nav/web/auth.py

```diff
@@ -16,14 +15,124 @@
 """
 Contains web authentication functionality for NAV.
 """
+"""NAV authentication and authorization middleware for Django"""
```

Comment:
Three separate docstrings here seem silly. An unintended side-effect of squashing/rebasing, perhaps?

Reply:
Yep, couldn't think of anything better in a hurry. also check the wonderful first commit-message ;)